Cargo.lock

```diff
@@ -0,0 +1,639 @@
+# This file is automatically @generated by Cargo.
```

Comment:
proposal to remove `Cargo.lock` since this is a lib. Also adding it to be ignored in the `.gitignore` file

Reply:
done!